src/components/canrawview/canrawview_p.h

```diff
@@ -90,17 +121,41 @@ class CanRawViewPrivate : public QWidget {
     CanRawView* q_ptr;
     int prevIndex = 0;
     int rowID = 0;
+    QStringList columnsOrder;
+
+    void writeColumnsOrder(QJsonObject& json) const
+    {
+        uint ii = 0;
+        QJsonArray columnList;
+        for (auto iter : columnsOrder) {
+            if (ui->tv->isColumnHidden(ii++) == false) {
+                columnList.append(iter);
+            }
+        }
+        json["Columns"] = columnList;
```

Comment:
I used std::move for unique pointer, but for this case I never use it. If will be more efficient, why not. Changed.